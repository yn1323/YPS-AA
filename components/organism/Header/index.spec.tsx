import { render } from 'enzyme'
import React from 'react'
import { act } from 'react-test-renderer'
import Component from '.'
import TestWrapper from '@test/TestWrapper'

test(Component.name, async () => {
  let component
  await act(async () => {
    component = await render(
      <TestWrapper>
        <Component isLoggedIn />
      </TestWrapper>
    )
  })
  expect(component).toMatchSnapshot()
  let component2
  await act(async () => {
    component2 = await render(
      <TestWrapper>
        <Component isLoggedIn={false} />
      </TestWrapper>
    )
  })
  expect(component2).toMatchSnapshot()
})
