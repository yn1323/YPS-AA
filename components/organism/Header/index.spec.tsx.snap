// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header 1`] = `
<header
  class="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
>
  <div
    class="MuiToolbar-root MuiToolbar-regular Header__SpacedToolbar-sc-1u3zo5s-0 dJZBxm MuiToolbar-gutters"
    space="200"
  >
    <div
      style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
    >
      <div
        style="box-sizing:border-box;display:block;max-width:100%"
      >
        <img
          alt=""
          aria-hidden="true"
          role="presentation"
          src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzIiIGhlaWdodD0iMzIiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
          style="max-width:100%;display:block;margin:0;border:none;padding:0"
        />
      </div>
      <noscript>
        &lt;img srcSet="/_next/image?url=%2Fimages%2Flogo.png&amp;w=32&amp;q=75 1x, /_next/image?url=%2Fimages%2Flogo.png&amp;w=64&amp;q=75 2x" src="/_next/image?url=%2Fimages%2Flogo.png&amp;w=64&amp;q=75" decoding="async" style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%" class="HeaderTitle__ImageLogo-sc-nwr99g-1 gqIWiZ"/&gt;
      </noscript>
      <img
        class="HeaderTitle__ImageLogo-sc-nwr99g-1 gqIWiZ"
        decoding="async"
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
      />
    </div>
    <h6
      class="MuiTypography-root HeaderTitle__TitleText-sc-nwr99g-0 dfYmdX MuiTypography-h6"
    >
      タイトル
    </h6>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        Login
      </span>
    </button>
  </div>
</header>
`;

exports[`Header 2`] = `
<header
  class="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
>
  <div
    class="MuiToolbar-root MuiToolbar-regular Header__SpacedToolbar-sc-1u3zo5s-0 clVRoG MuiToolbar-gutters"
    space="20"
  >
    <div
      style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
    >
      <div
        style="box-sizing:border-box;display:block;max-width:100%"
      >
        <img
          alt=""
          aria-hidden="true"
          role="presentation"
          src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzIiIGhlaWdodD0iMzIiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
          style="max-width:100%;display:block;margin:0;border:none;padding:0"
        />
      </div>
      <noscript>
        &lt;img srcSet="/_next/image?url=%2Fimages%2Flogo.png&amp;w=32&amp;q=75 1x, /_next/image?url=%2Fimages%2Flogo.png&amp;w=64&amp;q=75 2x" src="/_next/image?url=%2Fimages%2Flogo.png&amp;w=64&amp;q=75" decoding="async" style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%" class="HeaderTitle__ImageLogo-sc-nwr99g-1 gqIWiZ"/&gt;
      </noscript>
      <img
        class="HeaderTitle__ImageLogo-sc-nwr99g-1 gqIWiZ"
        decoding="async"
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
      />
    </div>
    <h6
      class="MuiTypography-root HeaderTitle__TitleText-sc-nwr99g-0 dfYmdX MuiTypography-h6"
    >
      タイトル
    </h6>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        Login
      </span>
    </button>
  </div>
</header>
`;
